pandas/io/formats/style.py

```diff
@@ -1821,35 +1887,36 @@ def hide_columns(self, subset: Subset | None = None) -> Styler:
 
         >>> midx = pd.MultiIndex.from_product([["x", "y"], ["a", "b", "c"]])
         >>> df = pd.DataFrame(np.random.randn(6,6), index=midx, columns=midx)
-        >>> df.style.format("{:.1f}").hide_columns()
-        x   d    0.1    0.0    0.4    1.3    0.6   -1.4
-            e    0.7    1.0    1.3    1.5   -0.0   -0.2
-            f    1.4   -0.8    1.6   -0.2   -0.4   -0.3
-        y   d    0.4    1.0   -0.2   -0.8   -1.2    1.1
-            e   -0.6    1.2    1.8    1.9    0.3    0.3
-            f    0.8    0.5   -0.3    1.2    2.2   -0.8
+        >>> df.style.format("{:.1f}").hide_columns()  # doctest: +SKIP
+            a    0.1    0.0    0.4    1.3    0.6   -1.4
+        x   b    0.7    1.0    1.3    1.5   -0.0   -0.2
+            c    1.4   -0.8    1.6   -0.2   -0.4   -0.3
+            a    0.4    1.0   -0.2   -0.8   -1.2    1.1
+        y   b   -0.6    1.2    1.8    1.9    0.3    0.3
+            c    0.8    0.5   -0.3    1.2    2.2   -0.8
```

Comment:
good spot with the 'a, b, c'. But I prefer the 'x' and 'y' to be top aligned, as per the original.

Reply:
Why do you prefer the `x` and `y` aligned on the top if in the results they don't appear like that?

As you can see in the image `x` and `y` are aligned with `b`:
![image](https://user-images.githubusercontent.com/11354911/127569876-ecfa2bc8-1c98-419e-8265-81712429764e.png)

Reply:
for three reasons:
1) it defaults to the middle row in HTML because it is merged to a single cell and the css attribute 'vertical-align' is set to 'middle' by default, but this does not have to be the case, one might set it to 'top' for example.
2) if the example was and even number of elements (a,b,c,d) you would not be able to replicate this graphically since the 'middle' value would be split between two rows, and you would have to choose one which would be confusing.
3) the main reason is that this is a textual representation, not a graphical, and the documentation is most instructive when there are fewer artifacts to query. By placing the value at the top then it is more likely a user will assume the level_0 index headers have been merged to encompass (a,b,c).